docs/app/pages/components/components-sections/tables/column-picker/column-picker.component.html

```diff
@@ -53,6 +56,29 @@
     </tr>
 </uxd-api-properties>
 
+<uxd-api-properties tableTitle="ColumnPickerActionsContext Interface">
+    <tr uxd-api-property name="addSelection" type="ReadonlyArray<string>">
```

Comment:
Should this be `type="() => ReadonlyArray<string>"`? Also perhaps the function should be named `getAddSelection`?

Reply:
This is a direct reference to the array itself rather than it being a function as it is much better for change detection than having to call a function each time to check for changes

Reply:
Understood, thanks.